### ISSUE TITLE
basecontroller not working after upgrade to SDK 6.0.0.GA

### ISSUE THREAD

**Issue by MahmoudElmoghazy** [FokkeZB/nl.fokkezb.form#34]
Edit by @hazemkhaled
After Appc support NodeJS style require [[TIMOB-16078](https://jira.appcelerator.org/browse/TIMOB-16078)], basecontroller with ../ in path not working or maybe basecontroller into the widget not working

```
[DEBUG] Adding footerView from xml to this form
[ERROR] TiExceptionHandler: (main) [244,26067] ----- Titanium Javascript Runtime Error -----
[ERROR] TiExceptionHandler: (main) [0,26067] - In ti:/module.js:303,2
[ERROR] TiExceptionHandler: (main) [0,26067] - Message: Uncaught Error: Requested module not found: alloy/controllers/../widgets/nl.fokkezb.form/controllers/field
[ERROR] TiExceptionHandler: (main) [0,26067] - Source: 	throw new Error("Requested module not found: " + request); // TODO Set 'code' property to 'MODULE_NOT_FOUND' to match Node?
[ERROR] V8Exception: Exception occurred at ti:/module.js:303: Uncaught Error: Requested module not found: alloy/controllers/../widgets/nl.fokkezb.form/controllers/field
```

**Comment by HazemKhaled** [comment]
Alloy version?

**Comment by HazemKhaled** [comment]
Confirmed in SDK 6 with Android only on production, i think related to NodeJS require [[TIMOB-16078](https://jira.appcelerator.org/browse/TIMOB-16078)]

debugging now

**Comment by HazemKhaled** [comment]
```
[DEBUG] Adding footerView from xml to this form
[ERROR] TiExceptionHandler: (main) [244,26067] ----- Titanium Javascript Runtime Error -----
[ERROR] TiExceptionHandler: (main) [0,26067] - In ti:/module.js:303,2
[ERROR] TiExceptionHandler: (main) [0,26067] - Message: Uncaught Error: Requested module not found: alloy/controllers/../widgets/nl.fokkezb.form/controllers/field
[ERROR] TiExceptionHandler: (main) [0,26067] - Source: 	throw new Error("Requested module not found: " + request); // TODO Set 'code' property to 'MODULE_NOT_FOUND' to match Node?
[ERROR] V8Exception: Exception occurred at ti:/module.js:303: Uncaught Error: Requested module not found: alloy/controllers/../widgets/nl.fokkezb.form/controllers/field
```

**Comment by HazemKhaled** [comment]
So it happens in production and development also, just disable liveview and error appears

**Comment by HazemKhaled** [comment]
So workaround for now move field to your controller directory, till someone refactor the base controller code in this widget or Appc fix require.

Steps:
1. Copy field controller files to yout app
cp app/widgets/nl.fokkezb.form/controllers/field.js app/controllers
cp app/widgets/nl.fokkezb.form/views/field.js app/views
cp app/widgets/nl.fokkezb.form/styles/field.js app/styles

2. Replace path in the widget or any custom field you have
old exports.baseController = '../widgets/nl.fokkezb.form/controllers/field';
new exports.baseController = 'field';

**Comment by HazemKhaled** [comment]
JIRA ticket created
https://jira.appcelerator.org/browse/AC-4661

**Comment by hansemannn** [comment]
Hey guys, please try [this patch](https://github.com/appcelerator/titanium_mobile/pull/8673) for your 6.0.0.GA SDK. It will likely be included in 6.0.1 next week as well, thx for reporting!
